credentialMixin.js

```diff
@@ -58,7 +58,10 @@ export const credentialMixin = {
       const fields = {};
       // order fields based on schema
       Object.keys(this.schema).map(k => {
-        fields[k] = tmpFields[k];
+        if(tmpFields[k]) {
```

Comment:
This check is too broad and will cause values such as `0` to not display. Can this be constricted to only check for `null` or `undefined`. It is debatable whether or not we would want to prevent displaying the empty string.

Reply:
@gannan08 I have updated it to only check for null & undefined. I just wanted to point out that the value type is set to string here https://github.com/digitalbazaar/bedrock-vue-credential-card/blob/master/CredentialCardField.vue#L139, so everything we pass had to be converted to string. However doing that doesn't work for the address value since it's an object and so we will still have a vue warning when we pass an object. Let me know if you've got any suggestions on how I can solve that issue.

Reply:
prop types can have multiple types: https://vuejs.org/v2/guide/components-props.html

so use: [String, Object] as the type and thanks for finding that.